src/lib/mina_base/transaction_logic.ml

```diff
@@ -1262,6 +1262,8 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
            ; call_data = _ (* This is for the snapp to use, we don't need it. *)
            ; sequence_events
            ; depth = _ (* This is used to build the 'stack of stacks'. *)
+           ; use_full_commitment =
+               _ (* This is for the snapp to use, we don't need it. *)
```

Comment:
You'll probably want to use this below to toggle incrementing the nonce, and similarly so in transaction snark.